sql/updates/world/3.3.5/2016_xx_xx_xx_world_thorim.sql

```diff
@@ -0,0 +1,833 @@
+(10312,12,1,0,'');
+
+UPDATE `gameobject_template` SET `faction`=94, `flags`=16, `size`=3 WHERE `entry` IN (194312,194313,194314,194315);
+
+DELETE FROM `creature` WHERE `id` IN (32882,32908,32907,32885,32883,32886); -- PreAdds
+DELETE FROM `creature` WHERE `id` IN (32874,32875,33110); -- Colossus, Giant Adds
+
+-- Fix Thorim Controller multi spawns
+SET @CGUID:=136446;
+DELETE FROM `creature` WHERE `id`=32879;
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `MovementType`) VALUES
+(@CGUID, 32879, 603, 3, 1, 2134.774, -262.3073, 428.6936, 1.343904, 7200, 0, 0); -- 32879 (Area: 0) (Auras: 62184 - 62184)
+
+UPDATE `gameobject_loot_template` SET `LootMode`=1 WHERE `Entry`=27074;
+DELETE FROM `gameobject_loot_template` WHERE `Entry` IN (26955,26956);
+INSERT INTO `gameobject_loot_template` (`Entry`,`Item`,`Reference`,`Chance`,`QuestRequired`,`LootMode`,`GroupId`,`MinCount`,`MaxCount`,`Comment`) VALUES
+(26955,1,34372,100,0,1,0,1,1,'Thorim25 HM - normal mode loot'),
+(26955,2,12033,100,0,1,0,1,2,'Thorim25 HM - T8.5 tokens'),
+(26955,3,34154,10,0,1,0,1,1,'Thorim25 HM - Random Ulduar craft recipe'),
+(26955,45038,0,18,0,1,0,1,1,'Thorim25 HM - Val''anyr fragment'),
+(26955,45087,0,10,0,1,0,1,1,'Thorim25 HM - Runed Orb'),
+(26955,45470,0,0,0,1,1,1,1,'Thorim25 HM - Wisdom''s Hold'),
+(26955,45471,0,0,0,1,1,1,1,'Thorim25 HM - Fate''s Clutch'),
+(26955,45472,0,0,0,1,1,1,1,'Thorim25 HM - Warhelm of the Champion'),
+(26955,45473,0,0,0,1,1,1,1,'Thorim25 HM - Embrace of the Gladiator'),
+(26955,45474,0,0,0,1,1,1,1,'Thorim25 HM - Pauldrons of the Combatant'),
+(26955,45570,0,0,0,1,1,1,1,'Thorim25 HM - Skyforge Crossbow'),
+(26955,45817,0,100,1,1,0,1,1,'Thorim25 HM - Thorim''s Sigil'),
+(26955,47241,0,100,0,1,0,1,1,'Thorim25 HM - Emblem of Triumph'),
+
+(26956,1,34372,100,0,1,0,1,1,'Thorim25 - normal mode loot'),
+(26956,2,12033,100,0,1,0,1,2,'Thorim25 - T8.5 tokens'),
+(26956,3,34154,10,0,1,0,1,1,'Thorim25 - Random Ulduar craft recipe'),
+(26956,45038,0,8,0,1,0,1,1,'Thorim25 - Val''anyr fragment'),
+(26956,45087,0,10,0,1,0,1,1,'Thorim25 - Runed Orb'),
+(26956,47241,0,100,0,1,0,1,1,'Thorim25 - Emblem of Triumph');
+
+-- Pathing for Thorim Event Bunny Entry: 32892 'TDB FORMAT'
+SET @NPC := 136515;
+SET @PATH := @NPC * 10;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=2125.243,`position_y`=-279.0981,`position_z`=419.6088 WHERE `guid`=@NPC;
+DELETE FROM `creature_addon` WHERE `guid`=@NPC;
+INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '');
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@PATH,1,2125.243,-279.0981,419.6088,0,0,0,0,100,0),
```

Comment:
these sniffed waypoints still needs to be cleaned up. Like said above, some points are missing a in a few cycles. 

This should be replaced by  a single rotation.